lib/semian/sysv_state.rb

```diff
@@ -0,0 +1,31 @@
+require 'forwardable'
+
+module Semian
+  module SysV
+    class State < Semian::Simple::State #:nodoc:
+      include SysVSharedMemory
+      extend Forwardable
+
+      SYM_TO_NUM = {closed: 0, open: 1, half_open: 2}.freeze
+      NUM_TO_SYM = SYM_TO_NUM.invert.freeze
```

Comment:
Then there's going to be the question for why it's there, since it seems out of place in a class that hardcodes them. Then it leads to `why is it hardcoded` and stuff happens.